commit/plugin_functions.go

```diff
@@ -35,17 +35,18 @@ func observeLatestCommittedSeqNums(
 	sort.Slice(knownSourceChains, func(i, j int) bool { return knownSourceChains[i] < knownSourceChains[j] })
 	latestCommittedSeqNumsObservation := make([]plugintypes.SeqNumChain, 0)
 	if readableChains.Contains(destChain) {
-		lggr.Debugw("reading latest committed sequence from destination")
-		onChainLatestCommittedSeqNums, err := ccipReader.NextSeqNum(ctx, knownSourceChains)
+		lggr.Debugw("reading sequence numbers", "chains", knownSourceChains)
```

Comment:
Lets include as much context into the log as possible, maybe add `destChain` as well into the fields?

```suggestion
		lggr.Debugw("reading offramp min seq nrs", "destChain", destChain, "knownSourceChains", knownSourceChains)
```

As for the offramp address, I think we can include that as part of the logger's `Name` so it'll always be logged, so no need to include it here probably.